### PR TITLE
[DO NOT MERGE] DROOLS-1798: Show RuleModel imports in FromAccumulate FactType selector

### PR DESCRIPTION
**Pull request by jomarko**
Ensemble with:
- kiegroup/kie-wb-common#1306
- kiegroup/kie-soup#17

### CODE REVIEW

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
@jomarko I tried your change locally.

Whilst ```Number``` is listed as a data-type; selecting it results in the editor becoming disabled.

drools-1798

I'd expect the editor to not be disabled?

**Comment by jomarko** [comment]
@manstis totally right, I will notice you once solved this.

**Comment by jomarko** [comment]
@manstis The problem you have reported is gone, however now the Expression editor doesn't offer anything like `intValue()` etc. Could you please point me where to look?

**Comment by jomarko** [comment]
Working on tests

File: drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/test/java/org/drools/workbench/screens/guided/rule/client/widget/FromAccumulateCompositeFactPatternWidgetTest.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+    private FromAccumulateCompositeFactPatternWidget fromAccumulateWidget;
+
+    @Before
+    public void setUp() throws Exception {
+        GwtMockito.useProviderForType(ListBox.class, aClass -> listBox);
+
+        ruleModel = new RuleModel();
+        factTypes = new String[2];
+        factTypes[0] = "Person";
```
Comment (triceo) [inline drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/test/java/org/drools/workbench/screens/guided/rule/client/widget/FromAccumulateCompositeFactPatternWidgetTest.java]: If this is supposed to be the same `Person` as on line 93, please make it a constant and share it. Don't expect people to understand what you mean, make the code say what you mean.

File: drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/test/java/org/drools/workbench/screens/guided/rule/client/widget/FromAccumulateCompositeFactPatternWidgetTest.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    private ListBox listBox;
+
+    private FromAccumulateCompositeFactPatternWidget fromAccumulateWidget;
+
+    @Before
+    public void setUp() throws Exception {
+        GwtMockito.useProviderForType(ListBox.class, aClass -> listBox);
+
+        ruleModel = new RuleModel();
+        factTypes = new String[2];
```
Comment (triceo) [inline drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/test/java/org/drools/workbench/screens/guided/rule/client/widget/FromAccumulateCompositeFactPatternWidgetTest.java]: Why not `new String[] {"Person", "Car"};`?

Comment (triceo) [reply drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/test/java/org/drools/workbench/screens/guided/rule/client/widget/FromAccumulateCompositeFactPatternWidgetTest.java]: Also, why not assign the value on line 48, where you declare the field?

**Comment by jomarko** [comment]
Replaced by #720

**Pull request closed without merging**